easybuild/easyconfigs/l/libMemcached/libMemcached-1.0.18-GCC-6.4.0-2.28.eb

```diff
@@ -0,0 +1,20 @@
+easyblock = 'ConfigureMake'
+
+name = 'libMemcached'
+version = '1.0.18'
+
+homepage = 'https://memcached.org'
+description = """libMemcached is an open source C/C++ client library and tools for the memcached server (http://danga.com/memcached). It has been designed to be light on memory usage, thread safe, and provide full access to server side methods."""
+
+toolchain = {'name': 'GCC', 'version': '6.4.0-2.28'}
+toolchainopts = {'optarch': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['https://launchpad.net/libmemcached/%(version_major_minor)s/%(version)s/+download/']
+
+sanity_check_paths = {
+    'files': ['lib/libmemcached.so'],
```

Comment:
please use `SHLIB_EXT` rather than hardcoding `.so` (on macOS, it's `.dylib`, not `.so`)

```python
'files': ['lib/libmemcached.%s' % SHLIB_EXT],
```